### DISCUSSION TITLE
75.75.75.75

### DISCUSSION THREAD

**Discussion by iriedready** [will-stone/browserosaurus#531] [Q&A]
Hi Will,

Thanks for the app and for all your efforts, I really like the app and it is very helpful.  I would like to know why there is a request to 75.75.75.75 when you open the app?  Is that just to check for updates?

Thanks
D

**Comment by will-stone** [comment]
Hi. Presumably, yeah. Comment out this line (https://github.com/will-stone/browserosaurus/blob/c7d6d471682dd9f5bcc32d86bc59555440407b1e/src/main/state/middleware.action-hub.ts#L124) and re-package, then see if it’s still calling out.
